### PR TITLE
config: import vocab readers, transformers, writers consistently

### PR DESCRIPTION
**Pull request by yashlamba**
Fixes https://github.com/inveniosoftware/invenio-vocabularies/issues/348

### CODE REVIEW

File: invenio_app_rdm/__init__.py [hunk 1]
```diff
@@ -17,6 +17,6 @@
 #
 # See PEP 0440 for details - https://www.python.org/dev/peps/pep-0440
 
-__version__ = "13.0.0b0.dev8"
+__version__ = "13.0.0b0.dev9"
```
Comment (jrcastro2) [inline invenio_app_rdm/__init__.py:20]: I am bit confused, wasn't dev9 already released? https://github.com/inveniosoftware/invenio-app-rdm/commit/d4088f6f551799ba328a47171d76aa349b189088

Comment (yashlamba) [reply invenio_app_rdm/__init__.py:20]: Oh yes, I think the PR was older than that release, sorry. Updating.

**Review by jrcastro2 — APPROVED** [review]
LGTM! 🚀 

Not sure why the templates copyright gets updated but I guess we have updated it.

**Comment by yashlamba** [comment]
> LGTM! 🚀
> 
> Not sure why the templates copyright gets updated but I guess we have updated it.

Wait until you see the magical script by @slint

**Pull request merged**